dev/com.ibm.ws.jpa.tests.jpa_31_fat/test-applications/jpa31/src/io/openliberty/jpa/tests/jpa31/web/TestNewQueryTimeFunctionsServlet.java

```diff
@@ -602,12 +602,12 @@ public void testEXTRACTFunction_OnQueryResults_JPQL() throws Exception {
         }
 
         // Verify EXTRACT(SECOND) from a LocalDateTime field
-        if (!isSQLServer() && !isDB2ForLUW()) { // TODO, see https://github.com/eclipse-ee4j/eclipselink/issues/1573
-            q = em.createQuery("SELECT EXTRACT(SECOND FROM qdte.localDateTimeData) FROM QueryDateTimeEntity qdte WHERE qdte.id = 1");
-            result = q.getSingleResult();
-            Assert.assertNotNull(result);
-            Assert.assertEquals(0.0, result);
-        }
+//        if (!isSQLServer() && !isDB2ForLUW()) { // TODO, see https://github.com/eclipse-ee4j/eclipselink/issues/1573
+        q = em.createQuery("SELECT EXTRACT(SECOND FROM qdte.localDateTimeData) FROM QueryDateTimeEntity qdte WHERE qdte.id = 1");
+        result = q.getSingleResult();
+        Assert.assertNotNull(result);
+        Assert.assertEquals(0.0d, result);
+//        }
```

Comment:
Should these comments be removed now that the issues/TODOs are resolved?

Reply:
Probably should, yeah.